HerPortal/Models/HomepageViewModel.cs

```diff
@@ -58,6 +58,7 @@ public HomepageViewModel(User user, IEnumerable<CsvFileData> csvFiles)
                     }
                 )
             )
+            .OrderBy(kvp => kvp.Key)
```

Comment:
Does the normal C# dictionary preserve order? Should we be using an `OrderedDictionary` or something instead?

Reply:
Ah, damn. Good catch, sadly the only dictionary-type thing that preserves insertion order is the `OrderedDictionary`, and it isn't generic! I'll have a look

Reply:
Usually these things work fine for small sets, and then break once you breach a certain threshold*. That might mean that in practice using a normal Dictionary is fine, especially if we never expect to have a very large list.

If it's not trivial, we could split out a 'proper' implementation into a separate ticket - this doesn't feel like the highest priority for beta.

*Stretching my memory a bit, bit IIRC a hash dictionary usually stores things in 'buckets' by hash, but each bucket is basically just a list - so while it's small enough to fit in one bucket you've basically just got a fancy list.

Reply:
@DanCorderSoftwire and for posterity: sadly the helper we pass this dictionary to expects specifically a `Dictionary<string, LabelViewModel>` not an `IDictionary` or anything, so we can't even write a generic wrapper for the `OrderedDictionary` and use that! Joe and I have discussed, and we're okay for alphabetical order to not be _guaranteed_ in this case (I imagine that, practically speaking, the implementation causes the dictionary to display in this order)